presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -187,6 +190,44 @@ public void shouldGetTableTypes()
         assertThat(result).contains(row("TABLE"), row("VIEW"));
     }
 
+    @Test(groups = {JDBC, SIMBA_JDBC})
+    public void testSqlEscapeFunctions()
+            throws SQLException
+    {
+        if (usingSimbaJdbcDriver()) {
+            // These functions, which are implemented within the Simba JDBC and ODBC drivers,
+            // translate standard SQL syntax to equivalent Presto syntax.  The translated SQL
```

Comment:
This is not standard SQL.  These functions and escape syntax are part of the ODBC standard, and therefore intended to be handled by the drivers, not the databases themselves.